survival_guide/3_operations.adoc

```diff
@@ -677,39 +690,51 @@ With a access/secret pair, you’ll need to add the secret:
 [source,yaml]
 ----
 s3:
-    enableS3: true
-    enableIAM: false
-    accessKeyName: aws-access-key
-    secretKeyName: aws-secret-key
-    logDirectory: s3a://[BUCKET-NAME]
+  enableS3: true
+  enableIAM: false
+  accessKeyName: aws-access-key
+  secretKeyName: aws-secret-key
+  logDirectory: s3a://[BUCKET_NAME]
 ----
 
 ===== Configuring Spark
 
-After the History Server has been set up, then the Fusion job-launcher deployment ConfigMap’s `application.yaml` key will need to be updated with these Spark settings so the driver and executors know where to write out their logs. In this example, we’re redirecting to a GCS bucket:
+After the History Server has been set up, then the config map for Fusion's job-launcher will need its `application.yaml` key will need to be updated with these Spark settings so the driver and executors know where to write out their logs.
```

Comment:
```suggestion
After starting the history server, we must update the config map for Fusion's job-launcher so it can write logs to the same location that Spark History Server is reading from.
```